src/components/Circles/Thread.tsx

```diff
@@ -297,16 +285,47 @@ const defaultStyles = createStyles('Circles.Thread', (theme) => ({
     flex: 0,
   },
   addCommentText: {
-    color: theme.colors.onPrimaryContainer,
-    backgroundColor: theme.colors.primaryContainer,
-    padding: theme.spacing.small,
-    borderRadius: 10,
-    borderWidth: 1,
-    overflow: 'hidden',
+    lineHeight: 16,
+    fontSize: 14,
+    width: '90%',
+    alignSelf: 'center',
   },
   createPostModalContainer: {
     height: 0,
   },
+  toolbarContainer: {
```

Comment:
TODO: Remove these styles since this was all consolidated within CreatePostToolbar